src/sql/azdata.proposed.d.ts

```diff
@@ -73,6 +96,9 @@ declare module 'azdata' {
 
 		export interface ICellMetadata {
 			connection_name?: string;
+			dotnet_interactive?: {
```

Comment:
I don't like having implementation-specific stuff like this in our public API. Is this not something that could be internal instead?

Reply:
These azdata interfaces get used internally, so if we want strong typing we'd need to include it here.

Reply:
I realize this might be a bigger change though to update a bunch of references. If you start this and it turns out to be more than a couple of lines then we can get this in and do that in a follow up PR so it's easier to review.

Reply:
Yeah, there's over 40 references to the nb namespace just in cell.ts.

Reply:
Alright, let's do this as a follow up task.

It's fine not to go and fix every instance of us using azdata stuff in core to begin with though - we can focus on at least fixing the ICellMetadata usage and then fix up others as we go along.

Reply:
Replaced the ICellMetadata usage in CellModel.